tests/pipeline/test_technical.py

```diff
@@ -0,0 +1,227 @@
+from abc import ABCMeta, abstractmethod
+
+import numpy as np
+from operator import itemgetter
+import pandas as pd
+from six import with_metaclass
+import talib
+from toolz import compose, excepts
+
+from zipline.lib.adjusted_array import AdjustedArray
+from zipline.lib.adjustment import Float64Add
+from zipline.pipeline import TermGraph
+from zipline.pipeline.data import USEquityPricing
+from zipline.pipeline.engine import SimplePipelineEngine
+from zipline.pipeline.term import AssetExists
+from zipline.pipeline.factors import BollingerBands
+from zipline.testing import ExplodingObject, parameter_space
+from zipline.testing.fixtures import WithAssetFinder, ZiplineTestCase
+from zipline.testing.predicates import assert_equal
+
+
+_meta = type('_', (type(ZiplineTestCase), ABCMeta), {})
```

Comment:
Since there's nothing dynamic to the type creation here, do you think it would be clearer as
```python
class ZiplineABCMeta(type(ZiplineTestCase), ABCMeta):
    pass
```